tailwind.css

```
@@ -1,2 +1,3 @@
-@import "tailwindcss";
-@source "./src/**/*.tsx";
+@tailwind base;
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)